### PR TITLE
Blank vs helloWorld app for ios and android

### PR DESCRIPTION
**Pull request by corinnekrych**
DO NOT MERGE
for Bob only for now
@edewit where should we add https://github.com/feedhenry-templates/helloworld-windows?
same as ios and android?

### CODE REVIEW

**Comment by corinnekrych** [comment]
@secondsun @danielpassos @edewit this the work done for RHMAP-3324 (templates for 3.7.0) could you please check I haven't missed any new templates.

@danielpassos maybe your PR on SAML should be merge with this one. wdyt? or you merge it to master and we rebase.

**Comment by secondsun** [comment]
@corinnekrych I'll import it into my studio today and give it a whirl.

File: global.json [hunk 1]
```diff
@@ -238,15 +262,57 @@
...
+      "description": "A FeedHenry blank project with one iOS Client App and a cloud instance with a single endpoint",
+      "type": "default",
+      "icon": "icon-apple",
+      "category": "Sample Projects",
+      "screenshots": [{
+        "url": "https://raw.githubusercontent.com/feedhenry-templates/blank-ios-app/master/screenshots/iosproject_container.png"
+      }],
+      "appTemplates": ["blank_native_ios_client", "hello_world_mbaas_instance"]
+    },{
+      "id": "ios_blank_cocoapods_project",
```
Comment (jasonmadigan) [inline global.json]: hey @corinnekrych - just doing a quick scan of this and other PRs for 3.7.0. I'm thinking the cocoapods template should probably land in 3.8.0 instead, right?

The rest of the PR looks fine

**Comment by corinnekrych** [comment]
@jasonmadigan 
To be able to test issue like https://issues.jboss.org/browse/RHMAP-1955 a cocoapods template is handy.
Let me check with @balunasj

**Comment by jasonmadigan** [comment]
@corinnekrych cool. Basically if this template works okay with the 3.6/3.7 diggers and isn't dependant on new stuff, this should be okay. I'll give it a spin in a cluster now and test.

File: global.json (hunk 1 above)
Comment (jasonmadigan) [inline global.json]: have tested this locally and on a single node instance (http://testing.jmadigan.skunkhenry.com) - built okay

**Comment by jasonmadigan** [comment]
@corinnekrych gave this a quick spin in my cluster - feel free to merge this

**Comment by jasonmadigan** [comment]
@corinnekrych the fhcap PR likely needs to be updated

**Pull request merged**